scrapers/freshmaker.py

```diff
@@ -0,0 +1,84 @@
+# SPDX-License-Identifier: GPL-3.0+
+from __future__ import unicode_literals
+import urllib3
+
+from scrapers.base import BaseScraper
+from purview.models.freshmaker import FreshmakerEvent, ContainerBuilds
+from scrapers.utils import retry_session
+from purview.utils.general import timestamp_to_datetime
+from purview import log
+
+urllib3.disable_warnings()
```

Comment:
Let's remove this because we don't want to have `verify=False` in our production code. I'll help you get around this.